rust/extractor/src/main.rs

```diff
@@ -65,16 +68,20 @@ fn extract(
         )
     });
 }
-fn main() -> anyhow::Result<()> {
-    let cfg = config::Config::extract().context("failed to load configuration")?;
+
+fn run_extractor(mut cfg: config::Config) -> anyhow::Result<()> {
     stderrlog::new()
         .module(module_path!())
-        .verbosity(1 + cfg.verbose as usize)
+        .verbosity(cfg.verbose as usize)
```

Comment:
Please make sure we don't print too much log output by default. Too much logging makes it impossible to make sense of log files, and may also fail DCA jobs.

Reply:
As far as I could see `INFO` prints are manageable. I think this was raised when `rust-analyzer` progress was using `info!` and was seriously spamming the logs. But since then that was downgraded to `trace!`, so I think we can bring verbosity back up to `INFO`.

Right now the only `info!` I can find are
* printing `Cargo.toml` manifests found (only once as far as I could tell, not once per file)
* printing the configuration at the start (now done on multiple lines, looked really useful)
* printing a successful `cargo check` on the qltests that ask for it